### DISCUSSION TITLE
About add edge

### DISCUSSION THREAD

**Discussion by wubo2180** [pyg-team/pytorch_geometric#4998] [Q&A] [answered]
Hi, I want to do edge add operation on my graph data. Is there any function to solve my problem?

 Best wish!

**Comment by rusty1s** [comment]
You can simply concatenate it to your `edge_index`:
```python
edge_index = torch.cat([edge_index, new_edges], dim=1)
```

**Comment by rusty1s** [comment, marked as answer]
You can simply concatenate it to your `edge_index`:
```python
edge_index = torch.cat([edge_index, new_edges], dim=1)
```

**Reply by wubo2180** [reply]
Ok, thanks!
